11-spatial-cv.Rmd

```diff
@@ -310,30 +310,30 @@ There are dozens of packages for statistical learning, as described for example
 Getting acquainted with each of these packages, including how to undertake cross-validation and hyperparameter tuning, can be a time-consuming process.
 Comparing model results from different packages can be even more laborious.
 The **mlr** package was developed to address these issues.
-It acts as a 'meta-package', providing a unified interface to popular supervised statistical learning techniques including classification, regression, survival analysis and clustering [@bischl_mlr:_2016]
-<!-- .^[ -->
+It acts as a 'meta-package', providing a unified interface to popular supervised statistical learning techniques including classification, regression, survival analysis and clustering [@bischl_mlr:_2016].
+<!-- ^[ -->
 <!-- We will solely focus on supervised learning techniques in this chapter. -->
 <!-- ] -->
 The standardized **mlr** interface is based on eight 'building blocks'.
 As illustrated in Figure \@ref(fig:building-blocks), these have a clear order.
 
-```{r building-blocks, echo=FALSE, fig.height=4, fig.width=4, fig.cap="Basic building blocks of the mlr package. Source: http://bit.ly/2tcb2b7. Permission to reuse this figure was kindly granted.", fig.scap="Basic building blocks of the mlr package."}
+```{r building-blocks, echo=FALSE, fig.height=4, fig.width=4, fig.cap="Basic building blocks of the mlr package. Source: http://bit.ly/2tcb2b7. (Permission to reuse this figure was kindly granted.", fig.scap="Basic building blocks of the mlr package.)"}
```

Comment:
@jannes-m I think the closing bracket is in a wrong place...

Reply:
Looks strange, I'll admit that, still, I think this is the right place. This is also the place recommended by the copyeditor.

Reply:
Oh no, you are right, this is wrong, I'll update immediately

Reply:
thanks, well-spotted!!

Reply:
@jannes-m I think the issue is that the opening bracket is in `fig.cap` but the closing one is in `fig.scap`. So the caption would look like this, I believe:

> Basic building blocks of the mlr package. Source: http://bit.ly/2tcb2b7. (Permission to reuse this figure was kindly granted.

Reply:
Looks fixed by https://github.com/Robinlovelace/geocompr/pull/354/commits/ea281f5aca7c8c1300778b0e635c25b4bb6e7d32